homeassistant/components/statistics/sensor.py

```diff
@@ -319,15 +319,8 @@ def should_poll(self):
     @property
     def extra_state_attributes(self):
         """Return the state attributes of the sensor."""
-        extra_attr = {}
-        if self._samples_max_age is not None:
-            extra_attr = {
-                STAT_AGE_COVERAGE_RATIO: self.attributes[STAT_AGE_COVERAGE_RATIO]
-            }
         return {
-            **extra_attr,
-            STAT_BUFFER_USAGE_RATIO: self.attributes[STAT_BUFFER_USAGE_RATIO],
-            STAT_SOURCE_VALUE_VALID: self.attributes[STAT_SOURCE_VALUE_VALID],
+            key: value for key, value in self.attributes.items() if value is not None
```

Comment:
The old check was redundant to line 445, the new one generalizes the idea to exclude `None` attributes